ext/pdo_pgsql/tests/bug_last_insert_id.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+currval() vs lastval()
+PDO PgSQL Bug #1134 [BUG] New record, PostgreSQL and the Primary key https://github.com/phalcon/cphalcon/issues/1134
+Fix PHP original PDO_PGSQL::lastInsertId https://github.com/doctrine/dbal/pull/2329
```

Comment:
Well, It's expected that both thats had the same result. So, they run toghether.